README.md

```diff
@@ -17,7 +17,7 @@ The following list is a brief summary of the features available in v2 pre-releas
 - 🎮 Minecraft integration
   - Display your Java edition or Bedrock servers statuses
   - Account verification using MCAssoc or the NamelessMC plugin
-  - [In-game plugin](https://www.spigotmc.org/resources/nameless-plugin-for-v2.59032)
+  - [In-game plugin](https://www.spigotmc.org/resources/59032)
```

Comment:
Can this change to `plugin.namelessmc.com`?

Reply:
The plugin will probably need to move away from spigot for the next update so this is a good idea